packages/app-config-writer/src/preview-config/ui5-yaml.ts

```diff
@@ -0,0 +1,291 @@
+import { basename, join } from 'path';
+import { createPreviewMiddlewareConfig } from '../variants-config/ui5-yaml';
+import { ensurePreviewMiddlewareDependency, extractUrlDetails, isValidPreviewScript } from './package-json';
+import { FileName, getAllUi5YamlFileNames, getWebappPath, readUi5Yaml } from '@sap-ux/project-access';
+import { getPreviewMiddleware, isFioriToolsDeprecatedPreviewConfig } from '../variants-config/utils';
+import { renameSandbox } from './preview-files';
+import type { CustomMiddleware } from '@sap-ux/ui5-config';
+import type { Editor } from 'mem-fs-editor';
+import type { FlpConfig, MiddlewareConfig as PreviewConfig } from '@sap-ux/preview-middleware';
+import type { PreviewConfigOptions } from '../types';
+import type { ToolsLogger } from '@sap-ux/logger';
+import type { ValidatedUi5ConfigFileNames, Package } from '@sap-ux/project-access';
+
+const DEFAULT_FLP_PATH = 'test/flp.html';
```

Comment:
This is already defined in `@sap-ux/preview-middleware`:
https://github.com/SAP/open-ux-tools/blob/a756c044dea0e4745a8be6bd6977aeff32348c90/packages/preview-middleware/src/base/config.ts#L79

If we want to avoid a `dependency` to `@sap-ux/preview-middleware`, we could add it as a type in `@sap-ux/preview-middleware` like

```typescript
export type DefaultFlpPath = '/test/flp.html';
export const DEFAULT_PATH: DefaultFlpPath = '/test/flp.html';
```
and import the type here. This has the advantage, if it is changed in the source module `@sap-ux/preview-middleware`, we get a build error here in `@sap-ux/preview-config`

Reply:
Done in af76444c78a09afb6921c1ed9b4323aa2182321a